package br.com.sicredi.pautavotacao.utils.exceptions;

public class CPFException extends BaseException{
    public CPFException() {
        super("Ocorreu um erro ao acessar o servi√ßo de consulta ao CPF");
    }
}
